src/components/Tag/Tag.scss

```
@@ -20,17 +20,63 @@ $icon-size: rem(16px);
     color: var(--p-text-disabled, color('ink', 'lightest'));
   }
 
-  .TagText {
-    overflow: hidden;
-    text-overflow: ellipsis;
-    white-space: nowrap;
-  }
+  &.clickable {
+    @include unstyled-button;
+    cursor: pointer;
+    padding: 0 spacing(tight);
+    background-color: color('sky');
+
+    // Start - Delete below me during newDesignLanguage rollout
+    &:hover,
+    &:focus,
+    &:active {
+      background: color('sky', 'dark');
+    }
+    // End - Delete above me during newDesignLanguage rollout
+
+    &:disabled {
+      cursor: default;
+      pointer-events: none;
```

(I forgot this part in the suggestion!)

```suggestion
      pointer-events: none;
      background: none;
```